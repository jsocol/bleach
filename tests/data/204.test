<img name="bar" id="foo">
