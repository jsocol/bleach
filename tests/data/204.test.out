<img id="foo" name="bar">
